awswrangler/s3/_read_parquet.py

```diff
@@ -391,6 +393,7 @@ def read_parquet(
     chunked: Literal[True],
     use_threads: Union[bool, int] = ...,
     parallelism: int = ...,
+    bulk_read_parquet: bool = ...,
```

Comment:
Should we just call it `bulk_read`? It's pretty obvious that it's for parquet from the name of the method